src/js/transforms.js

```diff
@@ -42,3 +42,35 @@ gpii.binder.transforms.stripEmptyString = function (value) {
 fluid.defaults("gpii.binder.transforms.stripEmptyString", {
     gradeNames: ["fluid.standardTransformFunction"]
 });
+
+/**
+ *
+ * Transform the value returned by jQuery.val for a single checkbox into a boolean.
+ *
+ * @param value {Array} - An array of values, either the value attribute of a ticked checkbox, or "on" if the checkbox has no value specified.
+ * @return {Boolean} - `true` if the first value is checked, `false`.
+ *
+ */
+gpii.binder.transforms.checkToBoolean = function (value) {
+    return fluid.get(value, 0) ? true : false;
+};
+
+fluid.defaults("gpii.binder.transforms.checkToBoolean", {
```

Comment:
Shouldn't these be registered as inverses of each other? I see that in your example both forward and inverse rules were specified manually, but ideally the framework would compute one from the other (I think actually in this case it probably just about could).

Reply:
Happy to try it out and report back.

Reply:
Done, and confirmed working with new tests.